lib/packaging/deb/repo.rb

```
@@ -98,12 +98,16 @@ def repo_creation_command(prefix, artifact_directory)
 Components: #{subrepo}
 Description: Apt repository for acceptance testing" >> conf/distributions ; )
 
-      # Create the repositories using reprepro. Since these are for acceptance
+      # Create the repositories using aptly. Since these are for acceptance
       # testing only, we'll just add the debs and ignore source files for now.
       #
-      cmd << "reprepro=$(which reprepro) ; "
-      cmd << %Q($reprepro includedeb $dist ../../#{prefix}deb/$dist#{Pkg::Config.apt_repo_name ? "/#{subrepo}" : ""}/*.deb ; popd ; done ; )
-      cmd << "popd ; popd "
+      cmd << "aptly=$(which aptly) ; "
```

should we add something in here so we don't end up failing silently if aptly isn't found? Or maybe something more substantial around this block as a whole, looks like a lot of opportunity for silent failures.